pepper/cli.py

```diff
@@ -88,8 +93,8 @@ def add_globalopts(self):
         #    help="Redirect the output from a command to a persistent data store")
 
         optgroup.add_option('--fail-if-incomplete', action='store_true',
-            dest='fail_if_minions_dont_respond',
-            help=textwrap.dedent('''\
+                            dest='fail_if_minions_dont_respond',
+                            help=textwrap.dedent('''\
             Return a failure exit code if not all minions respond. This option
```

Comment:
Please retain the same indentation here and the similar changes above and below.